src/components/Editor/EditorTab.tsx

```diff
@@ -0,0 +1,29 @@
+import React from "react";
+import "./EditorTab.css";
+
+interface EditorTabProps {
+  title: string;
+  selected: boolean;
+  onClick: () => void;
+}
+
+const EditorTab: React.FC<EditorTabProps> = ({ title, selected, onClick }) => {
+  return (
+    <div
+      onClick={onClick}
+      className={`tab${selected ? " active" : ""}`}
+      style={{
+        display: "flex",
```

Comment:
Why not stylesheet?
Also how a11y-friendly is this? Could be nice if you're able to navigate using tab

Reply:
It doesn't actually support a11y
and stylesheet doesn't work